### PR TITLE
micro tvm optimization

### PR DESCRIPTION
**Pull request by AugustineAurelius**

### CODE REVIEW

File: tvm/cell/dict.go [hunk 1]
```diff
@@ -503,20 +505,18 @@ func loadLabel(sz uint, loader *Slice, key *Builder) (uint, *Builder, error) {
...
 
-	bitsLen := uint(math.Ceil(math.Log2(float64(sz + 1))))
-
 	ln, err := loader.LoadUInt(bitsLen)
 	if err != nil {
 		return 0, nil, err
 	}
 
-	var toStore []byte
+	toStore := make([]byte, 1+int(ln)/8)
```
Comment (xssnick) [inline tvm/cell/dict.go:513]: It seems less optimal, 2 allocations instead of one, if it is just `toStore = bytes.Repeat` we alloc only once

Comment (AugustineAurelius) [reply tvm/cell/dict.go:513]: Agreed, I apologize, I'll correct it now.

File: tvm/cell/dict.go [hunk 2]
```diff
@@ -516,7 +516,7 @@ func loadLabel(sz uint, loader *Slice, key *Builder) (uint, *Builder, error) {
 		toStore = bytes.Repeat([]byte{0xFF}, 1+(int(ln)/8))
 	} else {
 		// N of zeroes
-		toStore = bytes.Repeat([]byte{0x00}, 1+(int(ln)/8))
+		toStore = bytes.Repeat([]byte{0xFF}, 1+(int(ln)/8))
```
Comment (xssnick) [inline tvm/cell/dict.go:519]: 0x00

Comment (xssnick) [reply tvm/cell/dict.go]: no worries, just typo :)

**Pull request merged**